bin/ch-run.c

```diff
@@ -212,11 +273,22 @@ void fix_environment(struct args *args)
    }
 
    // --set-env and --unset-env.
+   char* env_set[50];
```

Comment:
What is the significance of 50? Should this be a macro, e.g., `#define ENV_MAX`?